packages/icons/iconBlockList.ts

```diff
@@ -0,0 +1,52 @@
+import { IconData } from './processIcons';
```

Comment:
These icons all had no visual output - I'm assuming this check was made before making the previous list of available icons that ended up in the `png` folder?